sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/ApplicationGateway.java

```diff
@@ -202,6 +203,9 @@ public interface ApplicationGateway
      */
     Mono<WebApplicationFirewallPolicy> getWebApplicationFirewallPolicyAsync();
 
+    /** @return SSL policy of the application gateway */
```

Comment:
Add Javadoc description. This will be reported as warning in Java 18+

Reply:
Sure, I'll add it.